### PR TITLE
fix mask3d from file w/o space attrs

### PR DESCRIPTION
**Pull request by GoldenAnpu**

### CODE REVIEW

File: supervisely/geometry/mask_3d.py [hunk 1]
```diff
@@ -268,13 +268,19 @@ def from_file(figure, file_path: str):
...
+                x=mask3d_header["space origin"][0],
+                y=mask3d_header["space origin"][1],
+                z=mask3d_header["space origin"][2],
+            )
+            figure.geometry._space = mask3d_header["space"]
+            figure.geometry._space_directions = mask3d_header["space directions"]
+        except KeyError as e:
+            if str(e) in ("'space origin'", "'space'", "'space directions'"):
+                logger.debug(
+                    f"The Mask3D geometry for '{figure.parent_object.obj_class.name}' object doesn't contain optional space attributes"
```
Comment (mkolomeychenko) [inline supervisely/geometry/mask_3d.py:282]: file_path

File: supervisely/geometry/mask_3d.py [hunk 2]
```diff
@@ -268,13 +268,19 @@ def from_file(figure, file_path: str):
...
+        try:
+            figure.geometry._space_origin = PointVolume(
+                x=mask3d_header["space origin"][0],
+                y=mask3d_header["space origin"][1],
+                z=mask3d_header["space origin"][2],
+            )
+            figure.geometry._space = mask3d_header["space"]
+            figure.geometry._space_directions = mask3d_header["space directions"]
+        except KeyError as e:
+            if str(e) in ("'space origin'", "'space'", "'space directions'"):
```
Comment (mkolomeychenko) [inline supervisely/geometry/mask_3d.py:280]: move to variable list and use in warning

File: supervisely/geometry/mask_3d.py (hunk 1 above)
Comment (mkolomeychenko) [reply supervisely/geometry/mask_3d.py:282]: will try to use space attributes later from the volume during upload?

Comment (mkolomeychenko) [reply supervisely/geometry/mask_3d.py:282]: warn

File: supervisely/geometry/mask_3d.py [hunk 3]
```diff
@@ -296,10 +302,11 @@ def create_from_file(cls, file_path: str) -> Mask3D:
...
             geometry._space = mask3d_header["space"]
             geometry._space_directions = mask3d_header["space directions"]
-        except KeyError:
-            logger.debug(
-                "The Mask3D geometry created from the file does not contain private attributes"
-            )
+        except KeyError as e:
+            if str(e) in ("'space origin'", "'space'", "'space directions'"):
+                logger.debug(
+                    "The Mask3D geometry created from the file does not contain optional space attributes"
```
Comment (mkolomeychenko) [inline supervisely/geometry/mask_3d.py:308]: analogy

**Review by mkolomeychenko — CHANGES_REQUESTED** [review]

File: supervisely/geometry/mask_3d.py (hunk 1 above)
Comment (GoldenAnpu) [reply supervisely/geometry/mask_3d.py]: instead of file_path use figure ID

**Pull request merged**